plugins/hypervisors/xenserver/src/main/java/com/cloud/hypervisor/xenserver/resource/wrapper/xenbase/CitrixMigrateCommandWrapper.java

```diff
@@ -81,6 +81,13 @@ public Answer execute(final MigrateCommand command, final CitrixResourceBase cit
                 }
                 citrixResourceBase.migrateVM(conn, dsthost, vm, vmName);
                 vm.setAffinity(conn, dsthost);
+
+                final String result = citrixResourceBase.callHostPlugin(conn, "vmops", "destroy_network_rules_for_vm", "vmName", command.getVmName());
+                if (result == null || result.isEmpty() || !Boolean.parseBoolean(result)) {
```

Comment:
Have you tried using `BooleanUtils`?

Reply:
Good point @rafaelweingartner. I will update to

```
if (BooleanUtils.toBoolean(result)) {
    s_logger.debug(String.format("Removed network rules from source host [%s] for migrated vm [%s]", dsthost.getHostname(conn), command.getVmName()));
} else {
    s_logger.warn(String.format("Failed to remove network rules from source host [%s] for migrated vm [%s]", dsthost.getHostname(conn), command.getVmName()));                
}
```